routes/web.php

```diff
@@ -18,7 +18,9 @@
 Route::view('/', 'marketing.home');
```

Comment:
We seem to have this route twice - it's on line 45 as well. I don't think we need it twice